.binder/postBuild

```diff
@@ -3,5 +3,6 @@
 set -e
 
 npm install -g yarn
+yarn set version berry
```

Comment:
Actually, it [seems](https://yarnpkg.com/getting-started/install#updating-to-the-latest-versions) this is the more correct invocation.

```suggestion
yarn set version stable
```